daffodil-runtime1/src/main/scala/edu/illinois/ncsa/daffodil/processors/parsers/PState.scala

```diff
@@ -160,14 +156,16 @@ final class PState private (
   dataProcArg: DataProcessor,
   var delimitedParseResult: Maybe[dfa.ParseResult],
   tunable: DaffodilTunables) // Runtime tunables obtained from DataProcessor)
-  extends ParseOrUnparseState(vmap, diagnosticsArg, One(dataProcArg), tunable)
-  with FormatInfo {
+  extends ParseOrUnparseState(vmap, diagnosticsArg, One(dataProcArg), tunable) {
 
-  override def currentNode = Maybe(infoset)
-
-  def codingErrorAction: CodingErrorAction = decoder.malformedInputAction()
+  def parse1(parser: Parser): Unit = {
```

Comment:
Don't really need this method here. parse1 of parser is already a centralized place to be sure that the setProcessor and unwind thereof are done.